src/app/shared/object-list/my-dspace-result-list-element/claimed-search-result/claimed-declined-task-search-result/claimed-declined-task-search-result-list-element.component.ts

```diff
@@ -31,11 +30,6 @@ export class ClaimedDeclinedTaskSearchResultListElementComponent extends SearchR
    */
   public showSubmitter = true;
 
-  /**
-   * Represent item's status
-   */
-  public status = MyDspaceItemStatusType.DECLINED_TASk;
```

Comment:
Is this supposed to be entirely removed?  Or should it be replaced with `Context.MyDSpaceDeclined`?